site/content/docs/concepts/pipelines.en.md

```diff
@@ -95,7 +95,7 @@ stages:
 ```
 You can see every available configuration option for `pipeline.yml` on the [pipeline manifest](../manifest/pipeline.en.md) page.
 
-There are 3 main parts of this file: the `name` field, which is the name of your CodePipeline, the `source` section, which details the repository and branch to track, and the `stages` section, which lists the environments you want this pipeline to deploy to. You can update this anytime, but you must run `copilot pipeline update` afterwards.
+There are 3 main parts of this file: the `name` field, which is the name of your CodePipeline, the `source` section, which details the repository and branch to track, and the `stages` section, which lists the environments you want this pipeline to deploy to. You can update this anytime, but you must run `copilot pipeline update` (or `copilot pipeline deploy`) afterwards.
```

Comment:
Maybe
```suggestion
There are 3 main parts of this file: the `name` field, which is the name of your CodePipeline, the `source` section, which details the repository and branch to track, and the `stages` section, which lists the environments you want this pipeline to deploy to. You can update this anytime, but you must run `copilot pipeline deploy` (or `copilot pipeline update`) afterwards.
```

Reply:
I mostly made `deploy` primary and `update` secondary, but in this instance, this was specifically about changing the manifest later on and updating, not really about the initial deployment, so I let `update` be the primary command here. That's just my reasoning; I'm fine with changing it if you want me to!

Reply:
What do you think of mentioning only `copilot pipeline deploy` when it comes to initial deployment, and only `copilot pipeline update` when it comes to updating (so just mention the primary one)? 

We can mention that they are aliases to each other in the `pipeline-deploy-update` page, but not necessarily everywhere in the doc. 

Just a thought, definitely not an ask for change!

Reply:
Hiding `pipeline update` entirely, so changed that page accordingly.